azure-pipelines.yml

```diff
@@ -67,13 +68,60 @@ jobs:
           mayapy --version
           mayapy run_tests.py
         displayName: "Unittests"
+
       - script: |
           mayapy build_livedocs.py && mayapy test_docs.py
         displayName: "Test docs"
+
       - script: |
           mayapy build_docs.py
         displayName: "Build docs"
 
+      - publish: $(System.DefaultWorkingDirectory)/build/html
+        artifact: docs
+        condition: and(succeeded(), eq(variables['containerImage'], 'mottosso/maya:2020'))
+
+  # -----------------------------------------------------------------------
+  #
+  # Deploy docs
+  #
+  # -----------------------------------------------------------------------
+
+  - job: Deploy_Docs
+    condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
+    pool:
+      vmImage: "ubuntu-latest"
+    steps:
+      - task: DownloadPipelineArtifact@2
+        inputs:
+          artifact: docs
+          path: $(Build.SourcesDirectory)/build/html
+
+      - script: |
+          git init
+          git config --local user.name "Azure Pipelines"
+          git config --local user.email "mitchellcoote@gmail.com"
```

Comment:
Ah I see this is what you meant with `user.name`. This is fine, let it be known who accomplished this feat. :)